kinetic/distribution.yaml

```diff
@@ -9498,6 +9498,16 @@ repositories:
       url: https://github.com/appliedAI-Initiative/orb_slam_2_ros.git
       version: master
     status: maintained
+  orb_slam2_with_maps-odom:
+    doc:
+      type: git
+      url: https://github.com/Inspur-ROS/ORB_SLAM2_WITH_MAPS-ODOM.git
```

Comment:
is this a ROS package?
I don't see a package.xml here

Reply:
this package also seems to be a fork

Reply:
This package is based on orb_slam2 demo,we should cd ORB_SLAM2_With_Maps-Odom/Examples/ROS/ORB_SLAM2,there is a manifest.xml.

Reply:
mmm ... the top level `CMakeLists` is not from a ROS package, and the example you pointed out seems to be using a legacy rosbuild package manifest (which I'm not sure if catkin supports or not).

I don't know if non-ROS packages are accepted here or not. @cottsay can you chime in?

Reply:
Old rosbuild stuff we definitely don't support.  In certain circumstances, we can take in non-ROS (catkin or ament) packages, but they need special handling at release time.

But I do think this PR should be split up, there is just too much going on in a single request.  Usually we prefer one package per pull request.

Reply:
OK,I am trying to create a new PR. Regarding the issue of rosbuild, we will discuss it in the orb_slam2_with_maps_odom package PR.